ui/app/AppLayouts/Chat/ChatColumn.qml

```diff
@@ -307,14 +307,13 @@ Item {
                 visible: isBlocked
                 color: Style.current.red
                 opacity: 0.1
-            }
 
-            Text {
-                id: blockedText
-                anchors.centerIn: blockedBanner
-                visible: isBlocked
-                color: Style.current.red
-                text: qsTr("Blocked")
+                Text {
+                    id: blockedText
+                    anchors.centerIn: blockedBanner
+                    color: Style.current.red
+                    text: qsTr("Blocked")
+                }
```

Comment:
Could you confirm it is still working, I remember I had to make it that way otherwise the opacity of the text is also 0.1 while it should be 1

Reply:
I see, so we need a parent Item then